libr/debug/debug.c

```diff
@@ -640,8 +640,7 @@ R_API bool r_debug_select(RDebug *dbg, int pid, int tid) {
 R_API const char *r_debug_reason_to_string(int type) {
 	switch (type) {
 	case R_DEBUG_REASON_DEAD: return "dead";
-	case R_DEBUG_REASON_ABORT: return "abort";
```

Comment:
I think segfault should be generic reason

Reply:
I am not sure for what. For abort? Absolutely not. An exploit developper could not see if the challenge binary is not tricked.